src/app/child-dev-project/children/aser/model/aser.ts

```diff
@@ -58,40 +34,44 @@ export class Aser extends Entity {
     dataType: "configurable-enum",
     innerDataType: "reading-levels",
   })
-  hindi: ReadingLevel;
+  hindi: SkillLevel;
   @DatabaseField({
     label: $localize`:Label of the Bengali ASER result:Bengali`,
     dataType: "configurable-enum",
     innerDataType: "reading-levels",
   })
-  bengali: ReadingLevel;
+  bengali: SkillLevel;
```

Comment:
Is this a good time to move the explicit subject properties out of the class and into config?

In general, this would be a good case to be merged with the HistoricalEntityData module once we can define entity types completely in config.

Reply:
The aser is also where the HistoricalEntityData was inspired by. However, the color coding is unique to the aser.

Reply:
I know. Now that color coding is configurable maybe we wouldn't need the specific ASER implementation anymore?